tachyon/math/base/big_int.h

```diff
@@ -438,47 +435,24 @@ struct ALIGNAS(internal::LimbsAlignment(N)) BigInt {
     return *this;
   }
 
+  constexpr BigInt Add(const BigInt& other) {
+    uint64_t unused = 0;
+    return Add(other, unused);
+  }
+
+  constexpr BigInt Add(const BigInt& other, uint64_t& carry) {
+    BigInt ret;
+    DoAdd(*this, other, carry, ret);
+    return ret;
+  }
+
   constexpr BigInt& AddInPlace(const BigInt& other) {
     uint64_t unused = 0;
     return AddInPlace(other, unused);
   }
 
   constexpr BigInt& AddInPlace(const BigInt& other, uint64_t& carry) {
```

Comment:
```suggestion
  constexpr BigInt& AddInPlace(const BigInt& other, uint64_t carry) {
```
Can you make a separate commit fixing this since it's out of scope of this commit?